### PR TITLE
Add option to show piece notation to move list in analysis view

### PR DESCRIPTION
**Pull request by freefal**
Closes https://github.com/veloce/lichobile/issues/412

### CODE REVIEW

**Comment by veloce** [comment]
Thanks for that!

It would be even nicer if this was a global settings, under 'settings -> game display' entry probably, and would display the piece notation also in game replay (visible in landscape mode)

**Comment by freefal** [comment]
It is in 'settings -> game display'. I'll make game replay use it as well.
I never go into landscape so didn't know it was used elsewhere.

On Tue, Oct 11, 2016 at 3:38 AM, Vincent Velociter <notifications@github.com

**Comment by veloce** [comment]
One more location to change: offline games in landscape mode.

File: project/src/js/ui/shared/round/view/replay.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import * as helper from '../../../helper';
+import settings from '../../../settings';
```
Comment (veloce) [inline project/src/js/ui/shared/round/view/replay.js]: bad path

**Comment by veloce** [comment]
You could cache access to the pref since it's been read a lot during redraws and it's localstorage.

See how I do it in `Board.tsx` for pieceTheme and boardTheme as an example.

**Comment by freefal** [comment]
Apologies for the bad settings. I thought I tested everywhere I added the
pieceNotation, but clearly I didn't. Now fixed.
Thanks for the tip on caching. Now implemented.

On Sat, Oct 15, 2016 at 1:11 PM, Vincent Velociter <notifications@github.com

**Pull request merged**